src/djangosaml2_spid/conf.py

```diff
@@ -248,8 +248,9 @@ def config_settings_loader(request: Optional[HttpRequest] = None) -> SPConfig:
 
                 # When set to true, the SP will consume unsolicited SAML
                 # Responses, i.e. SAML Responses for which it has not sent
-                # a respective SAML Authentication Request.
-                'allow_unsolicited': False,
+                # a respective SAML Authentication Request. Set to True to
```

Comment:
please specicy why this is needed and only for test purpose

Reply:
From SAML 2.0 profiles document http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf:

4.1.5 Unsolicited Responses

An identity provider MAY initiate this profile by delivering an unsolicited <Response> message to aservice provider.An unsolicited <Response> MUST NOT contain an InResponseTo attribute, nor should any bearer<SubjectConfirmationData> elements contain one. If metadata as specified in [SAMLMeta] is used,the <Response> or artifact SHOULD be delivered to the <md:AssertionConsumerService> endpointof the service provider designated as the default.Of special mention is that the identity provider MAY include a binding-specific "RelayState" parameter thatindicates, based on mutual agreement with the service provider, how to handle subsequent interactionswith the user agent. This MAY be the URL of a resource at the service provider. The service providerSHOULD be prepared to handle unsolicited responses by designating a default location to send the useragent subsequent to processing a response successfully.

Reply:
So if there is a manner to check if the response is from an IdP then _allow_unsolicited_ could be dinamically set True if the requester is an IdP or False otherwise ...

Reply:
yes we know that's something for idp-initiated login but I meant a comment regarding the samesite cookie restrictions but if already in README we can move ahead with this